### PR TITLE
Fix: NodeTool on armv7l (Raspberry Pi)

### LINKED ISSUE #10295: NodeTool does not work on ARM
**Question, Bug, or Feature?**  
*Type*: Bug

**Enter Task Name**: NodeTool

## Environment
- Server - Azure Pipelines
- Agent - Private - Raspberry Pi 3b+

## Issue Description
```shell
##[section]Starting: Install Node.js
==============================================================================
Task         : Node.js Tool Installer
Description  : Finds or downloads and caches the specified version spec of Node.js and adds it to the PATH.
Version      : 0.149.0
Author       : Microsoft Corporation
Help         : 
==============================================================================
##[error]Unable to find Node version '8.x' for platform linux and architecture arm.
##[section]Finishing: Install Node.js
```

This is because `os.arch()` equals `arm` but valid downloads are one of `linux-arm64, linux-armv6l, linux-armv7l`.

### PR DESCRIPTION
**Pull request by timfish**
**Task name**: NodeToolV0

**Description**: Fixes #10295. Adds a special case as per @damccorm suggestion.

**Documentation changes required:**  No

**Added unit tests:** No

**Attached related issue:** #10295

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by timfish** [comment]
Any chance of getting this merged? 

Our builds have been showing this for 18 months now, simply because we're having to ignore this failure:

image

**Comment by egor-bryzgalov** [comment]
@timfish 
Could you please bump the task's version following [this guide?](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md)

**Comment by timfish** [comment]
@egor-bryzgalov all done

**Comment by egor-bryzgalov** [comment]
> @egor-bryzgalov all done

Thank you! Could you please update the version at the task.loc.json file also?

**Comment by timfish** [comment]
👍

File: Tasks/NodeToolV0/task.loc.json [hunk 1]
```diff
@@ -58,4 +58,4 @@
   "messages": {
     "ToolFailed": "ms-resource:loc.messages.ToolFailed"
   }
-}
+}
```
Comment (DaniilShmelev) [inline Tasks/NodeToolV0/task.loc.json:61]: NIT: looks like you've accidentally removed newline at the end of the file

**Review by DaniilShmelev — APPROVED** [review]

**Pull request closed without merging**